test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/RemoveRecordConnectorSpec.scala

```diff
@@ -85,4 +89,23 @@ class RemoveRecordConnectorSpec extends BaseConnectorSpec {
     result.left.value mustBe BadRequest("error")
   }
 
+  def expectedHeader: Seq[(String, String)] =
```

Comment:
This can probably go in the baseConnector spec as a function with an auth token and correlationId parameters. As I see it's repeated in all the connector tests but just the auth token changes

Reply:
yes, actually there is already an buildHeader in the BaseConnector that is exactly the same. I completaly miss that one. Thanks